api/src/main/java/org/openmrs/Changeable.java

```diff
@@ -38,4 +39,6 @@
 	 * @param dateChanged - the date the object was last changed
 	 */
 	void setDateChanged(Date dateChanged);
+
+	List<User> findPossibleValues(String searchText);
```

Comment:
We also don't need to make these changes. They are part of the `Attributable` interface.